content/articles/ms-ignite-2018.md

```diff
@@ -13,70 +13,42 @@ logo: /img/logo-sm.svg
 ![](http://appanacdn.blob.core.windows.net/cdn/icons/aic.png)
 
 ### Introduction
+Just three months ago [we announced that Microsoft joined the OpenCensus project](https://open.microsoft.com/2018/06/13/microsoft-joins-the-opencensus-project/), and last week at [Microsoft Ignite](https://www.microsoft.com/ignite) we were able to share a few more details about how OpenCensus will be leveraged and advanced.
 
-Just 3 month passed since the announcement that Microsoft is joining the
-OpenCensus project. This week at [Microsoft
-Ignite](https://www.microsoft.com/ignite) a Open Census were part of
-many conversation. It makes the goal of building a standardized platform
-for instrumenting metrics and distributed traces that work across
-various programming languages and technologies closer.
+### OpenCensus spotlight
 
-### Keynote
-
-As part of a keynote Microsoft announced support of Virtual Nodes by
-Azure Kubernetes Service. Virtual Nodes allow to scale your app faster,
-in a matter of minutes. What's the best way to demonstrate this if not
-the Application Insights [Live
+During one of Ignite's keynotes, Microsoft announced support for Azure Kubernetes Service (AKS) Virtual Nodes. Virtual Nodes allow applications to scale more quickly, in a matter of mere minutes. The demonstration included a highlight of Azure Monitor for Application's [Live
```

Comment:
"During one of Ignite's keynotes," weakens the message. Just say "During Ignite's keynote"?

Reply:
Fixed.